app/tests/usecases/test_revoke_exchange_command_handler.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import patch
+
+from hallebarde.usecases import revoke_exchange_command_handler
+
+
+class TestRevokeExchange:
+
+    @patch('hallebarde.usecases.revoke_exchange_command_handler.exchange_repository')
+    @patch('hallebarde.usecases.revoke_exchange_command_handler.file_repository')
+    def test_handle_should_delete_exchange_in_db(self, mock_file_repository, mock_exchange_repo):
+        # Given
+
+        # When
+        revoke_exchange_command_handler.handle(identifier='test_identifier')
```

Comment:
I would put the `test_identifier` init in the given, as you use it in both when & then